README.md

```diff
@@ -79,4 +80,4 @@ docker run -it -p 8080:80 --rm opendrinks
 
 ## License
 
-MIT
+The [MIT](https://opensource.org/licenses/MIT) License
```

Comment:
Can you link this to the LICENSE.md file instead? The opensource.org license looks like a placeholder.

Reply:
Yes, I can add it for sure